src/Form/Section.php

```diff
@@ -96,6 +96,12 @@ public function getQuestions(): array
                 $question = new Question();
                 $question->getFromResultSet($row);
                 $question->post_getFromDB();
+
+                if ($question->getQuestionType() === null) {
+                    // The question might belong to a disabled plugin
```

Comment:
Maybe add an entry in logs in that case? Or we're 100% sure this kind of issue can only came from a disabled plugin?
Anyway, maybe it should be precised somewhere for the form to be adjusted (or to enable again the faultly plugin).

Reply:
For now I've silently ignored any unknown type that may come from a plugin (question type, question type category, form destination, ...).

I guess it could still be logged (maybe a notice ?), I can add a backlog entry to do that if that is OK for everyone.

Reply:
Added to backlog: https://github.com/orgs/glpi-project/projects/7/views/1?pane=issue&itemId=56177382